.mergify.yml

```diff
@@ -36,7 +36,7 @@ pull_request_rules:
         - closed
       - and:
         - label=automation
-        - head~=^update-go-version
+        - head~=^updatecli
```

Comment:
name of the branch to be deleted, this is the prefix name for the updatecli automation.